src/emulator/storage/apis/gcloud.ts

```diff
@@ -28,58 +28,64 @@ export function createCloudEndpoints(emulator: StorageEmulator): Router {
     next();
   });
 
-  gcloudStorageAPI.get("/b/:bucketId/o/:objectId", (req, res) => {
-    const md = storageLayer.getMetadata(req.params.bucketId, req.params.objectId);
-
-    if (!md) {
-      res.sendStatus(404);
-      return;
-    }
+  gcloudStorageAPI.get(
+    ["/b/:bucketId/o/:objectId", "/download/storage/v1/b/:bucketId/o/:objectId"],
```

Comment:
The only change in this file is this line. The rest is caused by the prettier formatting.